administrator/components/com_guidedtours/src/Extension/GuidedtoursComponent.php

```diff
@@ -28,6 +28,78 @@ class GuidedtoursComponent extends MVCComponent implements BootableExtensionInte
 {
     use HTMLRegistryAwareTrait;
 
+    /**
+     * The step type
+     *
+     * @since __DEPLOY_VERSION__
+     */
+    public const STEP_TYPE_NAMES = [
+        self::STEP_NEXT   => 'COM_GUIDEDTOURS_FIELD_VALUE_NEXT_STEP',
+        self::STEP_REDIRECT => 'COM_GUIDEDTOURS_FIELD_VALUE_REDIRECT',
+        self::STEP_INTERACTIVE    => 'COM_GUIDEDTOURS_FIELD_VALUE_INTERACTIVE_STEP',
+    ];
+
+    /**
+     * A regular step.
+     *
+     * @since __DEPLOY_VERSION__
+     */
+    public const STEP_NEXT = 0;
+
+    /**
+     * A step that redirects to another page.
+     *
+     * @since __DEPLOY_VERSION__
+     */
+    public const STEP_REDIRECT = 1;
+
+    /**
+     * A step that allows interactions from the user.
+     *
+     * @since __DEPLOY_VERSION__
+     */
+    public const STEP_INTERACTIVE = 2;
+
+    /**
+     * The step type
+     *
+     * @since __DEPLOY_VERSION__
+     */
+    public const STEP_INTERACTIVETYPE_NAMES = [
+        self::STEP_INTERACTIVETYPE_FORM_SUBMIT   => 'COM_GUIDEDTOURS_TYPE_INTERACTIVE_STEP_FORM_SUBMIT_LABEL',
+        self::STEP_INTERACTIVETYPE_TEXT => 'COM_GUIDEDTOURS_TYPE_INTERACTIVE_STEP_TEXT_FIELD_LABEL',
```

Comment:
whatever the naming convention used for the keys it should be consistent

Reply:
Agreed